### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thelovekesh**
Fixed some typing and grammatical errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ By using [Packer's DigitalOcean Builder](https://www.packer.io/docs/builders/dig
...
 
-> The destination directory must already exist. If you need to create it, use a shell provisioner just prior to the file provisioner in order to create the directory. If the destination directory does not exist, the file provisioner may succeed, but it will have undefined results.
+> The destination directory must already exist. If you need to create it, use a shell provisioner just before the file provisioner to create the directory. If the destination directory does not exist, the file provisioner may succeed, but it will have undefined results.
 
 This configuration also uses Packer's [shell provisioner](https://www.packer.io/docs/provisioners/shell.html) to run scripts from the `/scripts` directory and update installed APT packages using an inline task.
 
-After making changes to the configuration, packer validate command.
+After making changes to the configuration, the packer validates the command.
```
Comment (bradford-hamilton) [inline README.md:55]: I think this may be slightly clearer: `After making changes to the configuration, run the packer validate command.`

**Review by bradford-hamilton — COMMENTED** [review]
Hey, thanks! Just one small comment.

File: README.md (hunk 1 above)
Comment (thelovekesh) [reply README.md:55]: I got it wrong maybe! Should I change this to this `After making changes to the configuration, run the packer validate command.`?

Comment (bradford-hamilton) [reply README.md:55]: Sure, sounds good to me.

**Comment by bradford-hamilton** [comment]
Awesome, thanks so much @thelovekesh!

**Pull request merged**